packages/projection/src/operators/Mappers/withHandles.ts

```diff
@@ -36,6 +41,36 @@ const assetIdToUTF8Handle = (assetId: Cardano.AssetId, cip67Asset: CIP67Asset |
   return assetNameToUTF8Handle(Cardano.AssetId.getAssetName(assetId));
```

Comment:
Please update `isValidHandle` implementation to return true for `sub@handle` format. It is not required for projection, but it is exported from `core` package and could be used in other contexts.

Reply:
addressed here: 461a2a90b5e16ca98d4b96448256d0506b930011 (I thought we would open a ticket for it after, as there was a previous closed PR for it).

Reply:
Feel free to open a ticket if you think it's out of scope of this. I think it still makes sense to do it in same PR as it's part of adding support for subhandles